app/Services/ParsableContent.php

```diff
@@ -33,10 +40,62 @@ public function __construct(private array $providers = [
         //
     }
 
+    /**
+     * Flushes the cache for the given content.
+     */
+    public static function flush(string $key, ?bool $all = null): void
```

Comment:
I wasn't sure on if you would want a seperate method for `flush` & `remove`. If you'd prefer, I can add in remove method & make flush just flush the entire cache, as you'd expect it to.

Reply:
make the key optional, when not provided, flush all.

Reply:
Done!